thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric/component.js

```diff
@@ -41,6 +41,7 @@ export default Component.extend({
     function() {
       const { selectedUrns, entities } = this.getProperties('selectedUrns', 'entities');
       const result = filterPrefix(selectedUrns, 'thirdeye:metric:')
+        .filter(urn => urn in entities)
```

Comment:
If you do this check here to confirm the urn is in `entitites`, then we can remove the condition check here for `entity`.

`const agg = entity ? { alias: entity.label, id: entity.urn.split(':')[2] } : {};`